content/src/Blip.Api.Template/MySettings.cs

```diff
@@ -12,6 +14,30 @@ public class MySettings
         /// </summary>
         [JsonProperty("BlipBotSettings")]
         public BlipBotSettings BlipBotSettings { get; set; }
+
+        [JsonProperty("ContentProvider")]
+        public ContentProviderSettings ContentProvider { get; set; }
+
+        [JsonProperty("SmallTalksConfig")]
+        public Dictionary<string, List<string>> SmallTalksConfig { get; set; }
+
+        [JsonProperty("NameConfig")]
+        public NameConfig NameConfig { get; set; }
+    }
+
+
+    public class NameConfig
+    {
+        [JsonProperty("CleanNameRegexPattern")]
+        public string CleanNameRegexPattern { get; set; }
+    }
+
+    public class ContentProviderSettings
+    {
```

Comment:
This should be removed alonside any mention of ContentProviderAPI, and moved to an independent package